src/components/Tile/Tile.js

```diff
@@ -464,9 +464,12 @@ export class ExpandableTile extends Component {
         {...other}
         onClick={this.handleClick}
         tabIndex={tabIndex}>
-        <button className={`${prefix}--tile__chevron`}>
+        <button
+          className={`${prefix}--tile__chevron`}
+          aria-labelledby="tileExpandIcon">
           {componentsX ? (
             <ChevronDown16
+              id="tileExpandIcon"
```

Comment:
Need a way to avoid ID duplication, if you search for [`uniqueId.js`](https://github.com/IBM/carbon-components-react/blob/v7.1.0/src/tools/uniqueId.js) references you'll see how other components in `carbon-cmoponents-react` solves this problem.